freebridge-ai/main.py

```
@@ -0,0 +1,66 @@
+class Query(BaseModel):
+    question: str
+
+@app.post("/ask")
+async def ask_question(query: Query):
+    if not rag_chain:
+        raise HTTPException(status_code=503, detail="RAG system is not initialized")
+    
+    try:
+        response = rag_chain.invoke({"input": query.question})
+        return {"answer": response["answer"]}
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)